src/main/java/reposense/git/GitUtil.java

```diff
@@ -79,15 +89,51 @@ static String convertToGitFormatsArgs(List<Format> formats) {
 
     /**
      * Returns the {@code String} command to specify the globs to exclude for `git log` command.
+     * Also checks that every file in {@code ignoreGlobList} is within the given repository {@code root}
      */
-    static String convertToGitExcludeGlobArgs(List<String> ignoreGlobList) {
+    public static String convertToGitExcludeGlobArgs(File root, List<String> ignoreGlobList) {
```

Comment:
need a few unit test on this method.

Reply:
Is it appropriate to put these unit tests in `GitLogTest`?

Reply:
Uh no, when the method under test is from the GitUtil, it would be best to put it under GitUtilTest.

Reply:
Noted. Have made the changes.